airbyte-webapp/src/views/Connection/CatalogDiffModal/components/DiffSection.tsx

```diff
@@ -0,0 +1,63 @@
+import { FormattedMessage } from "react-intl";
+
+import { AirbyteCatalog, StreamTransform } from "core/request/AirbyteClient";
+
+import { DiffVerb } from "../CatalogDiffModal";
+import { DiffHeader } from "./DiffHeader";
+import styles from "./DiffSection.module.scss";
+import { StreamRow } from "./StreamRow";
+
+interface DiffSectionProps {
+  streams: StreamTransform[];
+  catalog?: AirbyteCatalog;
+  diffVerb: DiffVerb;
+}
+
+export const DiffSection: React.FC<DiffSectionProps> = ({ streams, catalog, diffVerb }) => {
+  return (
+    <div className={styles.sectionContainer}>
+      <div className={styles.sectionHeader}>
+        <DiffHeader diffCount={streams.length} diffVerb={diffVerb} diffType="stream" />
+      </div>
+      <table>
```

Comment:
I have some concerns about using a table vs flex rows and columns. Tables tend to be more difficult to get the width to match consistently, and this is not necessarily rendering as a table, either.

Reply:
My thought around doing it as a table was accessibility.  Can get similar functionality with flex rows/columns and aria-labels if that seems better?

Reply:
It's fine to leave it with the table implementation until it becomes a problem.